packages/template-retail-react-app/app/pages/checkout/confirmation.jsx

```diff
@@ -70,8 +73,21 @@ const CheckoutConfirmation = () => {
 
     const submitForm = async (data) => {
         try {
-            await customer.registerCustomer(data)
+            const body = {
+                customer: {
+                    firstName: data.firstName,
+                    lastName: data.lastName,
+                    email: data.email,
+                    login: data.email
+                },
+                password: data.password
+            }
+            await register.mutateAsync(body)
+
+            navigate(`/account`)
         } catch (error) {
+            const json = await error.response.json()
```

Comment:
I think that this is assuming that the error is a `ResponseError` from the isomorphic SDK? But that's not guaranteed. (If, for example, not all data is passed, the SDK will throw just a standard `Error`.)